src/main/resources/application.yaml

```diff
@@ -21,7 +21,7 @@ queue:
 
 core_case_data:
   api:
-    url: 'http://localhost:4452'
+    url: ${CORE_CASE_DATA_API_URL:'http://localhost:4452'}
```

Comment:
do not add the env variable here, spring will convert to that already
that var is not here at all:
https://github.com/hmcts/cmc-claim-store/blob/master/src/main/resources/application.yml

Reply:
sure ... I would have thought it would create the property of .
core.case.data.api.url not core_case_data.api.url .... will try remove and re-try end-to-end